packages/core/drizzle/0004_woozy_sebastian_shaw.sql

```diff
@@ -0,0 +1,87 @@
+CREATE TABLE IF NOT EXISTS "latitude"."account" (
+	"userId" text NOT NULL,
+	"type" text NOT NULL,
+	"provider" text NOT NULL,
+	"providerAccountId" text NOT NULL,
+	"refresh_token" text,
+	"access_token" text,
+	"expires_at" integer,
+	"token_type" text,
+	"scope" text,
+	"id_token" text,
+	"session_state" text,
+	CONSTRAINT "account_provider_providerAccountId_pk" PRIMARY KEY("provider","providerAccountId")
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "latitude"."authenticator" (
```

Comment:
We don't need this one

Reply:
Hmm weird, it's [NextAuth's official docs for Drizzle](https://authjs.dev/getting-started/adapters/drizzle#schemas), but doesn't seem to be even mentioned on the rest of the docs.